buildpack.md

```diff
@@ -1035,6 +1036,7 @@ homepage = "<buildpack homepage>"
 clear-env = false
 description = "<buildpack description>"
 keywords = [ "<string>" ]
+sbom-formats = [ "<string>" ]
```

Comment:
Given that we now use `bom` everywhere else, should we also rename this to `bom-formats`?

Reply:
Please see 313fec2

Reply:
Please see a06a415